docs/migration.md

```diff
@@ -2,191 +2,188 @@
-3. Recreate the model instance after saving.
+```
+/var/lib/gpustack
+```
 
-!!! Note
+For other installation methods, refer to this [link](faq.md/#where-are-gpustacks-data-stored) to locate the data directory. In the following steps, this path is referenced as `${your-data-dir}`.
 
-    Distributed inference across multiple workers is currently not supported with custom inference backends.
+Due to architectural changes in v2.0.0, new components such as `postgres` and `s6-supervisor` may read from and write to files in `${your-data-dir}` using non-root users. Make sure that `${your-data-dir}` is accessible to all required users. You can use the following script to adjust permissions, but please be aware of the security risks and understand exactly what the script does before running it.
 
-## External Database Migration
+```bash
+dir="${your-data-dir}"
+chmod a+rx "${dir}/log" || true
+chmod a+rx $dir
+while [ "$dir" != "/" ]; do
+  chmod a+x "$dir"
+  dir=$(dirname "$dir")
+done
+```
 
-GPUStack supports using an external database to store the management data. If you previously deployed GPUStack with an external database, follow the steps below to migrate your data.
+### Migrate Server Using Docker
 
-!!! warning
+Since v2.0.0, you no longer need to specify the GPU computing platform or version (such as `-cpu`, `-cuda12.8`, or `-rocm`) for the server or worker images. Simply use the latest image: gpustack/gpustack:latest.
 
-      **Backup Required.** Before proceeding, back up the following:
+#### Embedded Database Migration (SQLite → PostgreSQL)
 
-      - The data directory used by your previous GPUStack installation, typically located at:
+In v0.7 and earlier, GPUStack used an embedded SQLite database by default to store management data. Starting from v2.0.0, GPUStack dropped SQLite support and now uses an embedded PostgreSQL database by default for improved performance and scalability.
 
-        ```
-        /var/lib/gpustack
-        ```
+Start the GPUStack with the `GPUSTACK_DATA_MIGRATION=true` to enable the embedded database migration. Replace `${your-data-dir}` with your legacy data directory containing the original SQLite database and related files:
+
+```bash
+sudo docker run -d --name gpustack \
+  --restart=unless-stopped \
+  --privileged \
+  --network=host \
+  --env GPUSTACK_DATA_MIGRATION=true \
+  --volume /var/run/docker.sock:/var/run/docker.sock \
+  --volume ${your-data-dir}:/var/lib/gpustack \
+  --runtime nvidia \
+  gpustack/gpustack:latest
+```
 
-      - Your external database, using the backup procedure recommended by your database system.
+!!! note
 
-### Migration Steps
+    Since v2.0.1, the environment variable `GPUSTACK_MIGRATION_DATA_DIR` introduced in v2.0.0 is deprecated. While setting it will still trigger database migration, its value will be ignored. It is recommended to use `GPUSTACK_DATA_MIGRATION=true` to indicate that the server requires database migration.
```

Comment:
It doesn't need migration from v2.0.0 to v2.0.1. Is the deprecation claim of GPUSTACK_MIGRATION_DATA_DIR needed?